R/facet-grid-.r

```diff
@@ -271,7 +278,7 @@ FacetGrid <- ggproto("FacetGrid", Facet,
       facet_rep <- rep(1:nrow(to_add), each = nrow(data))
 
       data <- unrowname(data[data_rep, , drop = FALSE])
-      facet_vals <- unrowname(cbind(
+      facet_vals <- unrowname(vec_cbind(
```

Comment:
Do you still need the `unrowname()`?

Reply:
I'm a bit unsure - I've kept it as a precaution but I'll investigate it a bit more

Reply:
turns out we do as base R reintroduces row names when indexing repeated rows 😞